src/services/http-service.tsx

```diff
@@ -0,0 +1,32 @@
+import { IProvider } from "../interfaces/iprovider";
+import { Observable } from "rxjs";
+import { ApiInfo } from './api-info';
+import axios from "axios";
+
+/**
+ * The purpose of this class is to serve as the base class for services that may consume REST APIs
```

Comment:
Same here

Reply:
I would say that it is a good practice to always have comments. Regardless if it seems a really straightforward code, I'd rather have comment than try to understand the code just looking into it.